tests/pipelines/test_pipelines_common.py

```diff
@@ -1373,18 +1375,56 @@ def test_sequential_cpu_offload_forward_pass(self, expected_max_diff=1e-4):
         output_without_offload = pipe(**inputs)[0]
 
         pipe.enable_sequential_cpu_offload()
+        assert pipe._execution_device.type == pipe._offload_device.type
 
         inputs = self.get_dummy_inputs(generator_device)
         output_with_offload = pipe(**inputs)[0]
 
         max_diff = np.abs(to_np(output_with_offload) - to_np(output_without_offload)).max()
         self.assertLess(max_diff, expected_max_diff, "CPU offloading should not affect the inference results")
 
+        # make sure all `torch.nn.Module` components (except those in `self._exclude_from_cpu_offload`) are offloaded correctly
```

Comment:
cc @DN6 
are these offload tests being run? e.g. I randomly checked on dit and all its offloading tests are failing but I did not see that in the reports (I looked in slow tests too and did not see any)

Reply:
I think they don't. Because here https://github.com/huggingface/diffusers/blob/f53352f750725a4bf4a44220db196c0f26f3ff81/.github/workflows/push_tests.yml#L53

pipeline-level module tests and not these:
https://github.com/huggingface/diffusers/blob/f53352f750725a4bf4a44220db196c0f26f3ff81/.github/workflows/push_tests.yml#L95

Will submit a PR to fix this.

Reply:
Could it be because DiT has low usage and hence it's not picked up by [`utils/fetch_torch_cuda_pipeline_test_matrix.py`](https://github.com/huggingface/diffusers/blob/main/utils/fetch_torch_cuda_pipeline_test_matrix.py) and ultimately not being included here? 

https://github.com/huggingface/diffusers/blob/b02e2113ff4625100a4412abd1ae0392ee415364/.github/workflows/push_tests.yml#L95

Reply:
Yeah, DiT has low usage so those slow tests are skipped. They should be in the nightly tests though.